Magefile.go

```diff
@@ -0,0 +1,34 @@
+//+build mage
+
+package main
+
+import (
+
+	// mage:import
+	_ "github.com/grafana/grafana-plugin-sdk-go/build"
+
+	"github.com/magefile/mage/sh"
+)
+
+// CheckCheck builds production back-end components.
+func CopyArtifacts() error {
+	if err := sh.RunV("mkdir", "-p", "dist"); err != nil {
+		return err
+	}
+
+	if err := sh.RunV("cp", "./README.md", "./dist"); err != nil {
+		return err
+	}
+
+	if err := sh.RunV("cp", "./src/plugin.json", "./dist"); err != nil {
+		return err
+	}
+
+	if err := sh.RunV("cp", "./src/pipeline.svg", "./dist"); err != nil {
+		return err
+	}
```

Comment:
@aknuds1 ... there must be something better than this!

Reply:
Will check.